### ISSUE TITLE
Russian language font issue (1177)

### ISSUE THREAD

**Issue by 6AKU66** [beyond-all-reason/spring#381]
https://www.youtube.com/watch?v=zqwz945Lc1I

[infolog.txt](https://github.com/beyond-all-reason/spring/files/9564957/infolog.txt)
Log from our testing with Ivand (a bit older version)

Windows 7
1177

**Comment by lhog** [comment]
Yep, I was able to find that the font fontconfig selects as substitution (mingliu.ttc) has wide spacing between the chars:

Compare:
https://cdn.discordapp.com/attachments/724924957074915358/1014215490559824013/unknown.png
https://cdn.discordapp.com/attachments/724924957074915358/1014215490996015124/unknown.png

Now the question is why it was selected for AKU and not selected for the majority of people.

**Comment by 6AKU66** [comment]
![Снимок10](https://user-images.githubusercontent.com/89547444/190147071-fc25f687-c1eb-45dc-aa04-e50ca9f76642.PNG)

Also maybe this message means something for you:
[t=00:00:01.855000][f=-000001] [Font] Warning: [CFontTexture] fonts/poppins-regular.otf has no kerning data
[t=00:00:01.868000][f=-000001] [Font] Warning: [CFontTexture] fonts/exo2-semibold.otf has no kerning data

**Comment by lhog** [comment]
Bisected issue to this commit https://github.com/beyond-all-reason/spring/commit/567528a98d1576df91985c479358d8d5e2bf94b7

**Comment by sprunk** [comment]
Game now has control over what fonts to pick after #1790 and other related work by Saurtron so I think this one is done, feel free to reopen if something is still missing engine-side.
